plugins/todo-backend/src/lib/TodoReader/TodoScmReader.ts

```diff
@@ -80,10 +81,25 @@ export class TodoScmReader implements TodoReader {
     { url }: ReadTodosOptions,
     etag?: string,
   ): Promise<CacheItem> {
+    const shouldNotInclude = [
```

Comment:
Make this a constant at the top of the file instead, if you cba it would be nice to have this list configurable as part of the `TodoScmReader`, as a second `options` argument to `fromConfig`.